utils/spacewalk-common-channels.ini

```diff
@@ -1469,6 +1469,73 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/SLE15-Uyuni-Client-Tools/SLE_15/
 
+[oraclelinux9]
+archs    = x86_64, aarch64
+name     = Oracle Linux 9 (%(arch)s)
+checksum = sha256
+gpgkey_url = 
```

Comment:
The doc [mentions](https://linux.oracle.com/security/gpg/) https://yum.oracle.com/RPM-GPG-KEY-oracle-ol9, but it returns 404, so I didn't add it.

Be aware, however that the GPG "file" contains in fact *two keys*:

```
[root@m211 ~]# gpg --show-keys /etc/pki/rpm-gpg/RPM-GPG-KEY-oracle 
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
pub   rsa4096 2022-01-19 [SC] [expires: 2042-01-14]
      3E6D826D3FBAB389C2F38E34BC4D06A08D8B756F
uid                      Oracle Linux (release key 1) <secalert_us@oracle.com>
sub   rsa4096 2022-01-19 [E] [expires: 2041-06-01]

pub   rsa4096 2022-01-19 [SC] [expires: 2042-01-14]
      982231759C7467065D0CE9B2A7DD07088B4EFBE6
uid                      Oracle Linux (backup key 1) <secalert_us@oracle.com>
sub   rsa4096 2022-01-19 [E] [expires: 2041-06-02]
```
AFAIK `spacewalk-common-channels` does *not* support two keys, so if they ever change to the backup key, we'll need to change it here, and at the DB with a SQL script.

Reply:
There is no oraclelinux:9 docker image yet. But guessing with the help of v8 we should try

`file:///etc/pki/rpm-gpg/RPM-GPG-KEY-oracle`

as URL. This is what we will do for all EL clones and point to the local installed GPG key.

Reply:
Still, you are not telling me to use this at `spacewalk-common-channels`, right? `file:///etc/pki/rpm-gpg/RPM-GPG-KEY-oracle` does not exist at the Server, even if I added it to the build-keys... or?

Reply:
No, it does not:
```
m225:~ # ls -l /etc/pki/rpm-gpg/RPM-GPG-KEY-oracle
ls: cannot access '/etc/pki/rpm-gpg/RPM-GPG-KEY-oracle': No such file or directory

Reply:
Clarified at slack, and changed.

Reply:
Just to write it down: this is the path on the client. The file is installed by oracle